### PR TITLE
🐦 Deleted .vs & .vscode files | Updated README 🐦 / Issue #37

### PR DESCRIPTION
**Pull request by Dmytrenko-Roman**

### CODE REVIEW

**Review by VsIG-official — APPROVED** [review]

**Review by mezgoodle — CHANGES_REQUESTED** [review]
Please change this content:

- [x] on [line 15](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L15) change `about AI on Python` to `about project on Python`
- [x] on [line 65](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L65) change `We get a description of the mood in _two different ways_: through a **bot** (Telegram) in the form of a text message and through a form on the **site** (Django). Next, the text is converted to a vector, compared with the song vectors in the database, and the result is returned as a song reference (in both cases).` to `We get a description of the mood in  _different ways_: through a **bot** (Telegram, Discord) in the form of a text message, through a form on the **site** (Django) , through a form on the **mobile app** (Kivy App). Next, the text is converted to a vector, compared with the mood vectors, getting a random song with this mood from databae and the result is returned as a song reference (in each cases).`
- [x] on [line 66](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L66) change `The site will have only one page with a form for entering a description. No other pages are planned yet.` to `The site will have one more page with a form for entering a song id from _Spotify_. This is for making database bigger. Also you can do it from bots in Telegram and Discord`
- [ ] Delete [line 71](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L71) and replace it with the next labels:  
  - `[Docker](https://github.com/mezidia/song-helper/actions/workflows/docker-publish.yml)`
  - `[Docker Image CI](https://github.com/mezidia/song-helper/actions/workflows/docker-image.yml)`

- [ ] on [line 94](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L94) add the next libraries:
  - `- [Kivy](https://kivy.org/#home)`
  - `- [pandas](https://pandas.pydata.org/)`
  - `- [discord.py](https://discordpy.readthedocs.io/en/stable/)`

**Comment by Dmytrenko-Roman** [comment]
> Please change this content:
> 
> * [x]  on [line 15](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L15) change `about AI on Python` to `about project on Python`
> * [x]  on [line 65](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L65) change `We get a description of the mood in _two different ways_: through a **bot** (Telegram) in the form of a text message and through a form on the **site** (Django). Next, the text is converted to a vector, compared with the song vectors in the database, and the result is returned as a song reference (in both cases).` to `We get a description of the mood in  _different ways_: through a **bot** (Telegram, Discord) in the form of a text message, through a form on the **site** (Django) , through a form on the **mobile app** (Kivy App). Next, the text is converted to a vector, compared with the mood vectors, getting a random song with this mood from databae and the result is returned as a song reference (in each cases).`
> * [x]  on [line 66](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L66) change `The site will have only one page with a form for entering a description. No other pages are planned yet.` to `The site will have one more page with a form for entering a song id from _Spotify_. This is for making database bigger. Also you can do it from bots in Telegram and Discord`
> * [x]  Delete [line 71](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L71) and replace it with the next labels:
>   
>   * `[Docker](https://github.com/mezidia/song-helper/actions/workflows/docker-publish.yml)`
>   * `[Docker Image CI](https://github.com/mezidia/song-helper/actions/workflows/docker-image.yml)`
> * [x]  on [line 94](https://github.com/mezidia/song-helper/blob/Delete-files/README.md#L94) add the next libraries:
>   
>   * `- [Kivy](https://kivy.org/#home)`
>   * `- [pandas](https://pandas.pydata.org/)`
>   * `- [discord.py](https://discordpy.readthedocs.io/en/stable/)`

Done! 😊

File: README.md [hunk 1]
```diff
@@ -62,13 +62,14 @@ The only functional requirement is to give a song
 ### Scenarios
 -----------
 
-- We get a description of the mood in _two different ways_: through a **bot** (Telegram) in the form of a text message and through a form on the **site** (Django). Next, the text is converted to a vector, compared with the song vectors in the database, and the result is returned as a song reference (in both cases).
-- The site will have only one page with a form for entering a description. No other pages are planned yet.
+- We get a description of the mood in _two different ways_: through a **bot** (Telegram) in the form of a text message and through a form on the **site** (Django). Next, the text is converted to a vector, compared with the song vectors in the database, and the result is returned as a song reference (in both cases). to We get a description of the mood in _different ways_: through a **bot** (Telegram, Discord) in the form of a text message, through a form on the **site** (Django) , through a form on the **mobile app** (Kivy App). Next, the text is converted to a vector, compared with the mood vectors, getting a random song with this mood from databae and the result is returned as a song reference (in each cases).
```
Comment (mezgoodle) [inline README.md:65]:
```suggestion
- We get a description of the mood in _different ways_: through a **bot** (Telegram, Discord) in the form of a text message, through a form on the **site** (Django) , through a form on the **mobile app** (Kivy App). Next, the text is converted to a vector, compared with the mood vectors, getting a random song with this mood from databae and the result is returned as a song reference (in each cases).
```

File: README.md [hunk 2]
```diff
@@ -62,13 +62,14 @@ The only functional requirement is to give a song
...
+- The site will have only one page with a form for entering a description. No other pages are planned yet. to The site will have one more page with a form for entering a song id from _Spotify_. This is for making database bigger. Also you can do it from bots in Telegram and Discord.
```
Comment (mezgoodle) [inline README.md:66]:
```suggestion
- The site will have one more page with a form for entering a song id from _Spotify_. This is for making database bigger. Also you can do it from bots in Telegram and Discord.
```

**Review by mezgoodle — CHANGES_REQUESTED** [review]
You forgot to delete old text

**Comment by Dmytrenko-Roman** [comment]
> You forgot to delete old text

Done! 😅

**Comment by mezgoodle** [comment]
> > You forgot to delete old text
> 
> Done! 😅

Alt Text

**Review by mezgoodle — APPROVED** [review]

**Pull request merged**